lib/edifunct/tokenizer.rb

```diff
@@ -6,9 +6,13 @@ module Edifunct
   # Tokenizer is responsible for splitting message into segments, data elements and components.
   class Tokenizer
     class << self
-      def for_message(_edifact_message)
-        # TODO: Should check if the message starts with `UNA`, and then extract the different separator/terminator settings to be used for initializing the tokenizer.
-        new
+      def for_message(edifact_message)
+        if edifact_message =~ /\AUNA/
+          # Example: UNA:+.? '
+          new(release_character: edifact_message[6], segment_terminator: edifact_message[8], data_element_separator: edifact_message[4], component_data_element_separator: edifact_message[3])
```

Comment:
I think it could be more clear which characters after `UNA` are what. I'm thinking something like this:

```ruby
component_data_element_separator, data_element_separator, _decimal_mark, release_character, _reserved, segment_terminator = edifact_message.slice(3, 6).split('')
```

It becomes a super long line so if it could be broken up that would be nice.